WMS ID: 11726

CDOracle opened the pull request:
# Add here your WMS ID 11726

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved


# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x ] All filenames are lower case (including folders, images, files, etc.)
- [x ] Filenames are descriptive
- [x ] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x ] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x ] Image references in markdown contain an alternate text

CDOracle commented:
Updating livelabs for an event on March 3rd

carmenberdant1 reviewed (changes requested):
Hi @CDOracle, please make sure your files are all in lowercase.

CDOracle commented:
My apologies.  I had removed those images from code but forgot to delete them.

There were 3 images that had uppercase and I corrected those images

Thank you

Casey

From: Carmen Berdant ***@***.***>
Sent: Thursday, February 27, 2025 3:27 AM
To: oracle-livelabs/oci ***@***.***>
Cc: Casey Doody ***@***.***>; Mention ***@***.***>
Subject: [External] : Re: [oracle-livelabs/oci] WMS ID: 11726 (PR #462)


@carmenberdant1 requested changes on this pull request.

Hi @CDOracle<https://urldefense.com/v3/__https:/github.com/CDOracle__;!!ACWV5N9M2RV99hQ!IcsDUJ0l9C1ubylkK-G4rKrTBb20dSnCPN5aUZ-jK_g1Ml-dY4KFZb6bTxdLyGSczx_v6e0vpjnT67Sv4y_yO8xW1RI$>, please make sure your files are all in lowercase.

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/oracle-livelabs/oci/pull/462*pullrequestreview-2647147411__;Iw!!ACWV5N9M2RV99hQ!IcsDUJ0l9C1ubylkK-G4rKrTBb20dSnCPN5aUZ-jK_g1Ml-dY4KFZb6bTxdLyGSczx_v6e0vpjnT67Sv4y_ytsR9XUY$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/BHBQXWLPG4MA22N37UVLJ532R3K7NAVCNFSM6AAAAABX6Z5JU2VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDMNBXGE2DONBRGE__;!!ACWV5N9M2RV99hQ!IcsDUJ0l9C1ubylkK-G4rKrTBb20dSnCPN5aUZ-jK_g1Ml-dY4KFZb6bTxdLyGSczx_v6e0vpjnT67Sv4y_yyta14F0$>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

CDOracle commented:
closing and resubmittinh

CDOracle commented:
closing and resubmitting
